_rules/image-accessible-name-23a2a8.md

```diff
@@ -3,7 +3,7 @@ id: 23a2a8
 name: Image has accessible name
 rule_type: atomic
 description: |
-  This rule checks that each image that is not marked as decorative, has an accessible name.
+  This rule checks that each image is either marked as decorative, or has an accessible name.
```

Comment:
I think that tweak significantly changes the meaning implied. The rule is not for checking that images are marked as decorative. I understand that needs to be worked out to check that non-decorative images have an accessible name.

It could change in a different way. Perhaps:
"This rule checks that each image has an accessible name, unless it is marked as decorative."

This states the intention of the rule up front, with the known exception following.